pkg/backingstore/backingstore.go

```diff
@@ -1116,7 +1113,6 @@ func MapNoobaaToBackingStores(noobaa types.NamespacedName) []reconcile.Request {
 			},
 		})
 	}
-	log.Infof("will reconcile all backingstores: %v", reqs)
```

Comment:
same here: 
`log.Infof("will reconcile these backingstores: %v due to changes with noobaa system %v", reqs, noobaa)`
also can you fix the typos in line 1105 and 1071 to Could (and not Cloud) and to secret and not secrte